vars/main.yml

```diff
@@ -1,2 +1,4 @@
 ---
 # vars file for supervisor
+
+
```

Comment:
I thought the standard is to put a blank space every after the last line. So, the 1st blank is for the 1st line (to be used) if in case the `/vars/main.yml` and the 2nd blank is for observing the standards.